.github/workflows/main.yml

```diff
@@ -26,3 +26,22 @@ jobs:
         with:
           name: ${{ runner.os }}
           path: build/distributions/*
+  release:
+    name: Create pubilc releases
+    runs-on: ubuntu-latest
+    needs: build
+    if: "startsWith(github.event.head_commit.message, 'Release')"
+    steps:
+      - name: get artefacts
+        uses: actions/download-artifact@v2
+      - name: get version from commit
+        run: echo "GITHUB_VERSION=$(echo ${{ github.event.commits[0].message }} | awk '{print $2}')" >> $GITHUB_ENV 
+      - name: test the var
+        run: echo $GITHUB_VERSION
+      - name: release public builds
+        uses: "marvinpinto/action-automatic-releases@latest"
+        with:
```

Comment:
Could you please make it create a draft release by default? Before releasing I'd like to modify the description to add more to the changelog and manually hit publish.